pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -98,6 +98,10 @@ func init() {
 					Verbs:     sets.NewString("update"),
 					Resources: sets.NewString("builds"),
 				},
+				{
+					Verbs:     sets.NewString("create"),
+					Resources: sets.NewString("builds/docker", "builds/source", "builds/custom"),
+				},
```

Comment:
@deads2k so are you saying that instead of gating updates on builds by type, we should ensure that you're not changing the type in the builds store? I agree, there's no reason to allow you to update the type.